### PR TITLE
add .script files for pypi package

### PR DESCRIPTION
**Pull request by Vaaceph**
This PR will add .script folder into PyPI package.

### CODE REVIEW

**Comment by alexeyegorov** [comment]
After some internal discussion we discovered the following: 
1. In our setup the scripts are only used locally. We switched the development to Docker and hence do not require the scripts. For the production deployment we also do not need them. 
2. In case we add the `.scripts` folder as suggested above, after the installation in the `.../lib/python3*/site-packages` you will finde two folders: `mara_app` and `.scripts` next to all the other packages installed through pip (e.g. pandas, numpy, etc.). We are not sure this would be an intended strategy. In case one would want to package those files together for PyPi or Conda, it may be a good solution to keep the scripts next to `static` folder inside the package.

**Comment by leo-schick** [comment]
I did a change to this PR which moves the .scripts folder into the PyPi package. After upgrading, one will have to copy the `mara_app/.scripts/init.mk` file to your project dir `.scripts/init.mk` once (if you use the .scripts logic).

@alexeyegorov does this look good to you?

File: mara_app/.scripts/init.mk [hunk 1]
```diff
@@ -7,8 +7,9 @@ SHELL=.scripts/mara-app/makeshell $(or $@,-)
 .SILENT:
 
 # copy scripts from mara-app package to project code
+.copy-mara-app-scripts: MODULE_LOCATION != .venv/bin/python -m pip show mara-app | sed -n -e 's/Location: //p'
 .copy-mara-app-scripts:
```
Comment (jankatins) [inline mara_app/.scripts/init.mk:11]: Is this intended that there are now two lines?

**Comment by alexeyegorov** [comment]
@leo-schick Thanks for the update. It looks good so far and we would test it in case you will be able to package and submit it to PyPi. :)

File: mara_app/.scripts/init.mk (hunk 1 above)
Comment (leo-schick) [reply mara_app/.scripts/init.mk:11]: Yes. This defines a local variable for the Makefile target .copy-mara-app-scripts. This is done that we don’t get into issues with other mara packages where maybe the same variable could be used.

**Pull request merged**